tools/windows/DatadogAgentInstaller/WixSetup/Datadog/AgentCustomActions.cs

```diff
@@ -11,9 +11,10 @@ public class AgentCustomActions
 
         public ManagedAction ProcessDdAgentUserCredentials { get; }
 
-
         public ManagedAction DecompressPythonDistributions { get; }
 
+        public ManagedAction ConfigureUser { get; }
+
         public AgentCustomActions()
         {
             // We need to explicitly set the ID since that we are going to reference before the Build* call.
```

Comment:
Github wouldn't let me add comment below.
Should we add `HideTarget()` to the `WriteConfig` CA because it has `APIKEY` and `PROXY_PASSWORD`?